examples/byoc-with-sagemaker-containers/serve-locally.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
I like the introduction of scripts to supplement many of the local mode, however I believe we should be steering towards using the Python SDK, especially with seeing your notebook example doing that.

Reply:
The introduction of the many bash scripts without explanation is a bit overwhelming.